### PR TITLE
Don't show the `Live Preview` button in the theme's modal window

### PR DESCRIPTION
**Pull request by anton-vlasenko**
Trac ticket: https://core.trac.wordpress.org/ticket/54578

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/themes.php [hunk 1]
```diff
@@ -307,9 +307,17 @@
...
+		'theme-editor.php',
+		'site-editor.php',
+		'edit.php?post_type=wp_navigation',
+	);
+
 	foreach ( (array) $submenu['themes.php'] as $item ) {
 		$class = '';
-		if ( 'themes.php' === $item[2] || 'theme-editor.php' === $item[2] || 0 === strpos( $item[2], 'customize.php' ) ) {
+
+		if ( in_array( $item[2], $forbidden_paths, true ) || 0 === strpos( $item[2], 'customize.php' ) ) {
```
Comment (costdev) [inline src/wp-admin/themes.php]:
```suggestion
		if ( in_array( $item[2], $forbidden_paths, true ) || str_starts_with( $item[2], 'customize.php' ) ) {
```
PHP 8 introduced `str_starts_with()` and we have polyfilled this in 5.9.

**Review by costdev — CHANGES_REQUESTED** [review]
Looks good! Just a minor suggestion to change `0 === strpos()` to `str_starts_with()`, which has now been polyfilled in 5.9.

File: src/wp-admin/themes.php (hunk 1 above)
Comment (anton-vlasenko) [reply src/wp-admin/themes.php]: Thank you for the review.
I've committed the suggested change.

**Review by SergeyBiryukov — APPROVED** [review]

**Review by hellofromtonya — APPROVED** [review]

**Comment by hellofromtonya** [comment]
Committed via https://core.trac.wordpress.org/changeset/52353.

**Pull request closed without merging**